assets/js/modules/ads/components/settings/SettingsForm.stories.js

```diff
@@ -99,7 +97,7 @@ export const GoogleTagGatewayDisabledWithWarning = Template.bind( null );
 GoogleTagGatewayDisabledWithWarning.storyName =
 	'GoogleTagGatewayDisabledWithWarning';
 GoogleTagGatewayDisabledWithWarning.scenario = {};
-GoogleTagGatewayDisabledWithWarning.args = {
+GoogleTagGatewayDisabledWithWarning.parameters = {
 	features: [ 'googleTagGateway' ],
 	setupRegistry: ( registry ) => {
```

Comment:
`features` belongs in parameters, but `setupRegistry` does not. Please keep in `args`.